public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/customs/guarantees
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/customs/guarantees
+    description: Production
+    variables: {}
+paths:
+  /customs/guarantees/:grn/balance:
+    post:
+      tags:
+        - customs
```

Comment:
The `tags` issue is an interesting one. For v1.0 of this API, we went with `ctc-guarantee-balance-api` but that was mainly because of Push Pull Notifications API using `push-pull-notification-api` (I haven't seen a specific guideline on Confluence about this).

When it comes to converting the Traders API from RAML to OAS, I imagine that we might want leeway to use the following `tags`:

- `Arrivals`
- `Departures`
- `Notifications` (or `Hooks` or `Webhooks`)